lib/block_view_validator.go

```diff
@@ -1883,20 +1871,6 @@ func (bav *UtxoView) JailAllInactiveValidators(blockHeight uint64) error {
 }
 
 func (bav *UtxoView) ShouldJailValidator(validatorEntry *ValidatorEntry, blockHeight uint64) (bool, error) {
-	// Return false if we haven't switched from PoW to PoS yet. Otherwise,
```

Comment:
This condition does not belong here. The caller should determine ahead of time if the block height is past the PoS consensus cutover. 

The role of this function should be simple: given a validator, determine if that validator is inactive based on the attributes of that validator.